### ISSUE TITLE
Metrics

### ISSUE THREAD

**Issue by purplefox** [spirit-labs/tektite#121]
We need to expose metrics from Tekite.

Prometheus format?

**Comment by devguyio** [comment]
I'd like to work on this

**Comment by purplefox** [comment]
Great!

I would suggest doing a bit of planning / design work for this first.

Things to think about?

* What metrics do we want?

* How will they be exposed? Prometheus?

**Comment by devguyio** [comment]
> Great!
> 
> I would suggest doing a bit of planning / design work for this first.
> 
> Things to think about?
> 
> * What metrics do we want?
> * How will they be exposed? Prometheus?

Yep, that's my plan. I'll create a proposal first and share before working. I've a good experience with Prometheus, metrics and Grafana.
